Library/ViewModels/SearchViewModel.swift

```diff
@@ -185,10 +186,27 @@ public final class SearchViewModel: SearchViewModelType, SearchViewModelInputs,
 
     self.scrollToProjectRow = self.transitionedToProjectRowAndTotalProperty.signal.skipNil().map(first)
 
-    // KSRAnalytics
+    // Tracking
 
-    viewWillAppearNotAnimated
-      .observeValues { AppEnvironment.current.ksrAnalytics.trackProjectSearchView() }
+    let projectsProperty = Property(
+      initial: [],
+      then: self.projects
+    )
+
+    let discoveryParamsProperty = Property(
+      initial: initialDiscoveryParams,
+      then: requestFirstPageWith
+    )
+
+    Property.combineLatest(
```

Comment:
Signal.combineLatest is something we use quite a bit and could possibly be applied here.